ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,1187 @@
+         */
+        relativeTo: 'plotBox',
+
+        /**
+         * Positioning for the button row. The breadcrumbs buttons
+         * will be aligned properly for the default chart layout
+         * (title,  subtitle, legend, range selector) for the custom chart
+         * layout set the position properties.
+         * @sample {highcharts} highcharts/breadcrumbs/position
+         *          Custom button positioning.
+         */
+        position: {
+            /**
+             * Horizontal alignment of the breadcrumbs buttons.
+             *
+             * @type {Highcharts.AlignValue}
+             */
+            align: 'left',
+
+            /**
+             * Vertical alignment of the breadcrumbs buttons.
+             *
+             * @type {Highcharts.VerticalAlignValue}
+             */
+            verticalAlign: 'top',
+
+            /**
+             * The X offset of the breadcrumbs button group.
+             */
+            x: 0,
+
+            /**
+             * The Y offset of the breadcrumbs button group.
+             */
+            y: 0
+        },
+
+        /**
+         * Options object for Breadcrumbs separator.
+         *
+         * @since     next
+         */
+        separator: {
+            /**
+             * A predefined shape or symbol for the separator. Other possible
+             *  values are `'circle'`, `'square'`,`'diamond'`.
+             * Custom callbacks for symbol path generation can also be added to
+             * `Highcharts.SVGRenderer.prototype.symbols`. The callback is then
+             * used by its method name.
+             *
+             * Additionally, the URL to a graphic can be given on this form:
+             * `'url(graphic.png)'`. Note that for the image to be applied to
+             * exported charts, its URL needs to be accessible by the export
+             * server. Recommended size of image is 10x10px.
+             *
+             * @type      {string}
+             * @since     next
+             */
+            symbol: 'triangle-right',
+            /**
+             * The size of the separator
+             *
+             * @type      {number}
+             * @since     next
+             */
+            size: 10
+        },
+
+        /**
+         * Show full path or only a single button.
+         *
+         * @type      {boolean}
+         * @since     next
+         * @sample {highcharts} highcharts/breadcrumbs/show-full-path
+         *          Show full path.
+         */
+        showFullPath: false,
+
+        /**
+         * CSS styles for all breadcrumbs.
+         *
+         * In styled mode, the breadcrumbs buttons are styled by the
+         * `.highcharts-range-selector-buttons .highcharts-button` rule with its
+         * different states.
+         *  @type {Highcharts.SVGAttributes}
+         *  @since     next
+         */
+        style: {
+            'fill': '#ffffff',
+            'position': 'absolute'
```

Comment:
Should be unnecessary. By default the button is SVG, otherwise the SVGRenderer should position it automatically.

Reply:
It looks to be unnecessary, I will remove it in the next commit if all tests pass.